build/pgbadger/Dockerfile

```diff
@@ -7,14 +7,16 @@ WORKDIR /go/src/github.com/crunchydata/crunchy-containers
 ADD ./badger ./badger
 RUN CGO_ENABLED=0 GOOS=linux go build -a -o badgerserver ./badger
 
-FROM ${PREFIX}/crunchy-pg-base:${BASEOS}-${PG_FULL}-${BASEVER}
+FROM ${PREFIX}/crunchy-base:${BASEOS}-${PG_FULL}-${BASEVER}
 
 # For RHEL8 all arguments used in main code has to be specified after FROM
 ARG BASEOS
 ARG DFSET
 ARG PACKAGER
 
 ARG PG_MAJOR
+# Needed due to lack of environment substitution trick on ADD
+ARG PG_LBL
```

Comment:
Why is this needed? I don't see it being used.

Reply:
Its being used to add the `crunchypg${PG_LBL}.repo` files here: https://github.com/CrunchyData/crunchy-containers/pull/1303/files/e37f4073d501710b7aa5338271223a204b58a565#diff-c56704700e72dcb6e7282ad4b8f427230f091b2a1f0c96bbd5b8a5402aea6a51R35

Reply:
Apparently I have an inability to read. Thanks.